.github/workflows/close_pr.yaml

```diff
@@ -1,17 +1,17 @@
-name: Pull Request
+name: Close Pull Request
 on:
   pull_request:
     types:
       - closed
-env:
-  HUMCTL_VERSION: '0.23.2'
+
 jobs:
   cleanup:
     runs-on: ubuntu-latest
     steps:
       - uses: humanitec/setup-cli-action@v1
         with:
-          version: ${{ env.HUMCTL_VERSION }}
+          version: ${{ vars.HUMCTL_VERSION }}
```

Comment:
If this is managed as variable, how can I safely test a humctl version bump? Right now I can just open a PR with the strings replaced, with a var its more complicated.

Reply:
Good practitioner's input, thanks - I pinned the version in the three workflows like previously.